image/image.go

```diff
@@ -468,7 +468,7 @@ func setupSeed(tsto *ToolingStore, model *asserts.Model, opts *Options, local *l
 		// required snaps like bluez which at this point
 		// still requires core will hang forever in seeding.
 		if strutil.ListContains(opts.Snaps, "core") {
```

Comment:
the check needs changing too,  opts.Snaps  can contain core (refering to the store) or a local core,  not quite sure right now what's best to check the latter